test/e2e/test-bento.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'Bento',
+  {
+    environments: ['single'],
+    fixture: 'bento/minimal.html',
+  },
+  async (env) => {
+    it('attaches shadow root without v0.js', async () => {
+      const shadowRoot = await env.controller.evaluate(
```

Comment:
Should we use a polling method with a timeout instead? If this fails...how long will it take?

Reply:
If there is a way to wait for the shadow root using a method akin to `MutationObserver` please enlighten me because I could not find it 🙂

I've updated the timeout so that it fails earlier.